app/src/main/java/com/example/ma/demomvp/screen/login/LoginContract.java

```diff
@@ -0,0 +1,14 @@
+package com.example.ma.demomvp.screen.login;
+
+public interface LoginContract {
+    interface LoginView{
+        void showProsesBar();
+        void hideProsesBar();
+        void setUserNameError();
```

Comment:
hàm set thường dùng để gán giá trị nào đó.
ở đây nếu em muốn show errror e có thể đặt tên method là
```suggestion
    void showUserNameError()
// hoặc 
     void onUserNameError()
```